tests/tuxlab.db.js

```diff
@@ -7,7 +7,8 @@ var async = require('async');
 
 //example databases
 var labs_good_1 = require('./example_data/labs/labs.good.js');
```

Comment:
It would be nice to convert this into an array of good and bad labs, and have comments explaining why each lab should fail